### ISSUE TITLE
Add missing lines of events translation

### ISSUE THREAD

**Issue by eadmaster** [eadmaster/pcrown#4]
Add missing lines of events translation, correct typos in the [old 2014 SamIAm script](https://github.com/eadmaster/pcrown/tree/master/script/eng/events).

Full list of files with untranslated lines:
````
000_00_1.TXT: Non-ISO extended-ASCII text, with very long lines (368)
000_00_8.TXT: Non-ISO extended-ASCII text, with very long lines (422)
000_02_6.TXT: Non-ISO extended-ASCII text
000_04_3.TXT: Non-ISO extended-ASCII text, with very long lines (607)
000_06_0.TXT: Non-ISO extended-ASCII text, with very long lines (2339)
000_07_1.TXT: Non-ISO extended-ASCII text
000_07_2.TXT: Non-ISO extended-ASCII text
000_07_3.TXT: Non-ISO extended-ASCII text
000_07_4.TXT: Non-ISO extended-ASCII text
000_07_5.TXT: Non-ISO extended-ASCII text
000_07_6.TXT: Non-ISO extended-ASCII text, with very long lines (769)
000_07_8.TXT: Non-ISO extended-ASCII text
000_07_9.TXT: Non-ISO extended-ASCII text
000_07_A.TXT: Non-ISO extended-ASCII text
000_07_B.TXT: Non-ISO extended-ASCII text
000_07_C.TXT: Non-ISO extended-ASCII text, with very long lines (385)
001_00_1.TXT: Non-ISO extended-ASCII text
009_00_0.TXT: Non-ISO extended-ASCII text, with very long lines (350)
015_00_1.TXT: Non-ISO extended-ASCII text, with very long lines (696)
026_00_0.TXT: Non-ISO extended-ASCII text, with very long lines (449)
026_01_0.TXT: Non-ISO extended-ASCII text, with very long lines (414)
026_01_1.TXT: Non-ISO extended-ASCII text, with very long lines (414)
026_01_2.TXT: Non-ISO extended-ASCII text, with very long lines (414)
026_01_3.TXT: Non-ISO extended-ASCII text, with very long lines (414)
031_02_0.TXT: Non-ISO extended-ASCII text, with very long lines (647)
035_11_0.TXT: Non-ISO extended-ASCII text
037_00_1.TXT: Non-ISO extended-ASCII text
045_00_3.TXT: Non-ISO extended-ASCII text, with very long lines (326)
057_28_0.TXT: Non-ISO extended-ASCII text, with very long lines (356)
061_00_1.TXT: Non-ISO extended-ASCII text, with very long lines (527)
061_00_2.TXT: Non-ISO extended-ASCII text, with very long lines (527)
071_21_0.TXT: Non-ISO extended-ASCII text, with very long lines (541)
071_21_1.TXT: Non-ISO extended-ASCII text
073_44_0.TXT: Non-ISO extended-ASCII text
073_44_1.TXT: Non-ISO extended-ASCII text, with very long lines (548)
074_00_1.TXT: Non-ISO extended-ASCII text, with very long lines (373)
074_02_1.TXT: Non-ISO extended-ASCII text, with very long lines (393)
079_03_0.TXT: Non-ISO extended-ASCII text, with very long lines (368)
090_00_0.TXT: Non-ISO extended-ASCII text, with very long lines (308)
093_15_0.TXT: Non-ISO extended-ASCII text, with very long lines (437)
101_14_0.TXT: Non-ISO extended-ASCII text
104_00_0.TXT: Non-ISO extended-ASCII text
107_04_0.TXT: Non-ISO extended-ASCII text
109_00_0.TXT: Non-ISO extended-ASCII text, with very long lines (312)
109_00_1.TXT: Non-ISO extended-ASCII text
110_00_1.TXT: Non-ISO extended-ASCII text, with very long lines (374)
112_00_0.TXT: Non-ISO extended-ASCII text
117_01_1.TXT: Non-ISO extended-ASCII text, with very long lines (389)
127_07_0.TXT: Non-ISO extended-ASCII text, with very long lines (428), with LF, NEL line terminators
135_21_0.TXT: Non-ISO extended-ASCII text
138_01_0.TXT: Non-ISO extended-ASCII text, with very long lines (592)
138_01_1.TXT: Non-ISO extended-ASCII text, with very long lines (592)
149_28_0.TXT: Non-ISO extended-ASCII text, with very long lines (458)
155_00_0.TXT: Non-ISO extended-ASCII text, with very long lines (435)
162_00_0.TXT: Non-ISO extended-ASCII text
166_00_0.TXT: Non-ISO extended-ASCII text, with very long lines (461)
176_00_0.TXT: Non-ISO extended-ASCII text
OP_EDOW.TXT:  Non-ISO extended-ASCII text, with very long lines (382)
OP_GRAD0.TXT: Non-ISO extended-ASCII text, with very long lines (506)
SELECT.TXT:   Non-ISO extended-ASCII text, with very long lines (688)
````

**Comment by MiYakuGaming** [comment]
I have a PR Push for it:
https://github.com/eadmaster/pcrown/pull/6

**Comment by eadmaster** [comment]
Thank you, these missing lines will be added in the next release

**Comment by eadmaster** [comment]
hi, i've just noticed you are still using the old scripts pre-splitted with
20 chars length, you should be using the original unsplitted version
instead.

I have them stored in this path now:
https://github.com/eadmaster/pcrown/tree/master/script/eng/events

**Comment by MiYakuGaming** [comment]
Thanks, I am still learning this all lol

**Comment by MiYakuGaming** [comment]
I've been using my fork to Translate existing Japanese lines and keeping the original English Lines intact.
You can see those PR's on my fork.

**Comment by eadmaster** [comment]
Ok, make sure you are in sync with my latest commit.

**Comment by MiYakuGaming** [comment]
I tried to do a PR of your commits but it seems the readme isn't doing well, not sure how to fix that at all.

**Comment by eadmaster** [comment]
All the missing lines are now translated, will be included in v0.6.
